.github/workflows/test.yml

```diff
@@ -2,9 +2,9 @@ name: test
 
 on:
   push: # Run on pushes to the default branch
-    branches: [main]
-  pull_request_target: # Also run on pull requests originated from forks
-    branches: [main]
+    branches: [main, 717-add-dbtrunner-local-executor-2] # TODO:remove before merge
+  # pull_request_target: # Also run on pull requests originated from forks # TODO: uncomment before merge
+    # branches: [main] # TODO: uncomment before merge
```

Comment:
Reminder to revert this before merging, stops workflow from running with the action file from main and only this branch.